src/main/java/core/basesyntax/FigureData.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureData {
+    String getData();
```

Comment:
```suggestion
    void draw();
```
Please, use `System.out.println()` in method implementation